addons/spreadsheet/static/src/o_spreadsheet/o_spreadsheet.js

```diff
@@ -4660,6 +4660,9 @@
 
     class ChartJsComponent extends owl.Component {
         static template = "o-spreadsheet-ChartJsComponent";
+        static props = {
+            figure: Object,
+        };
```

Comment:
This file is generated automatically in our repo (https://github.com/odoo/o-spreadsheet), the changes should be there and not it Odoo directly :) 

I can create the PR myself if you don't want to bother going in another repo :)

Reply:
All right ;)